tensorflow/contrib/factorization/python/ops/gmm_ops.py

```diff
@@ -36,13 +36,9 @@
 from tensorflow.python.ops import variable_scope
 from tensorflow.python.ops import variables
 from tensorflow.python.ops.embedding_ops import embedding_lookup
+from tensorflow.contrib.factorization.python.ops import constants
 from tensorflow.python.summary import summary
 
-# Machine epsilon.
-MEPS = np.finfo(float).eps
-FULL_COVARIANCE = 'full'
```

Comment:
@rmlarsen Just to confirm. Does it make sense if I am writing `FULL_COVARIANCE = "full"` instead of single quote declaration in `constants.py`? My CI build is failing.